include/ck/utility/math_v2.hpp

```diff
@@ -3,63 +3,113 @@
 
 #include <cmath>
 #include "data_type.hpp"
-#include "half.hpp"
+#include "type.hpp"
 
 namespace ck {
 namespace math {
 
-static inline __host__ float abs(float x) { return std::abs(x); };
+// math functions for the host,  some are implemented by calling C++ std functions
 
-static inline __host__ double abs(double x) { return std::abs(x); };
+static inline __host__ float ck_abs(float x) { return std::abs(x); };
 
-static inline __host__ int8_t abs(int8_t x)
+static inline __host__ double ck_abs(double x) { return std::abs(x); };
+
+static inline __host__ int8_t ck_abs(int8_t x)
 {
     int8_t sgn = x >> (8 - 1);
```

Comment:
I found some of the implementation in host and device are the same.
For example , __host__ int8_t ck_abs(int8_t x) and __device__ int8_t ck_abs(int8_t x)
Why do not you merge these function?
Just leave two implementations for those need to be different.